OpenFn/kit
# Runtime: adding more validation to workflow

## Short Description

Added functions to validate the workflow, steps and options and return errors or warnings accordingly

## Related issue

Fixes #681 

## Implementation Details

Added more validation functions to `validate-plan.ts` returning errors or warnings for a bad workflow structure.

## QA Notes

The following approach is not breaking any existing test cases. But we can add new test
cases where we can check the warning logs are printed successfully

## Checklist before requesting a review
